### ISSUE TITLE
Guest-specific operations were attempted on a machine that is not ready for guest communication.

### ISSUE THREAD

**Issue by Storm-Engineer** [vccw-team/vccw#162]
First time trying to use vccw or vagrant.

Trying to provision my site results in error. The only thing I've changed via site.yml is hostname and IP.

Console output: http://pastebin.com/cGfDNjkJ

**Comment by miya0001** [comment]
What is the version of the Vagrant.
Vagrant 1.8.5 has a bug.
Please use the latest version of the Vagrant.

https://github.com/vccw-team/vccw/issues/127

Thanks!

**Comment by Storm-Engineer** [comment]
Sorry for the slow reply. You were right, I still had 1.8.5 - I upgraded, now however I had a different issue: After "vagrant up" the process halts then times out. The site is still unreachable so I assume it didn't boot up.

```
==> webdev.wptesting: Waiting for machine to boot. This may take a few minutes...
    webdev.wptesting: SSH address: 127.0.0.1:2222
    webdev.wptesting: SSH username: vagrant
    webdev.wptesting: SSH auth method: private key

Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.
```

**Comment by miya0001** [comment]
Did you do `vagrant destroy` before `vagrant up`?
It looks there is an old machine.
Please open the VirtualBox app and remove all VCCW machines.

**Comment by Storm-Engineer** [comment]
Sorry for slow reply. YOu were right I had old version, I upgraded. But issue is not fixed!

I first try "vagrant up" - process halts and times out:

==> webdev.wptesting: Waiting for machine to boot. This may take a few minutes...
    webdev.wptesting: SSH address: 127.0.0.1:2222
    webdev.wptesting: SSH username: vagrant
    webdev.wptesting: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

Then I try "vagrant provision" - I get "Guest specific" error.

**Comment by Storm-Engineer** [comment]
Ah sorry, deleted that comment because it was inaccurate and wanted to write a new one just got distracted...

I first try "vagrant up" - process halts and times out:

==> webdev.wptesting: Waiting for machine to boot. This may take a few minutes...
    webdev.wptesting: SSH address: 127.0.0.1:2222
    webdev.wptesting: SSH username: vagrant
    webdev.wptesting: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

Then I try "vagrant provision" - I get "Guest specific" error.

I used "vagrant halt" after failed attempts and tried again, but same result. Now tried "vagrant destroy", same result.

I opened VirtualBox GUI, there are no machines left.

**Comment by miya0001** [comment]
Please upgrade VirtualBox to the latest version.

**Comment by Storm-Engineer** [comment]
I already have the latest (5.1.8)

**Comment by tmoffett-centrify** [comment]
I ran into this issue as well.  It was an issue with trying to run a 64-bit VM with VirtualBox.  I had to make sure virtualization was turned on in my BIOS.  So, if you don't have a host machine that supports virtualization, it may not work for you.

Some more info I found:
https://forums.virtualbox.org/viewtopic.php?f=1&t=62339
http://askubuntu.com/questions/41550/how-do-i-run-a-64-bit-guest-in-virtualbox
